coordinator/src/orderbook/trading.rs

```diff
@@ -119,12 +121,24 @@ impl Trading {
                     TradingMessage::NewOrder(new_order_msg) => {
                         // todo(holzeis): spawn a task here to not block other users from trading if
                         // this is taking some time.
-                        let result = self.process_new_order(new_order_msg.new_order).await;
+                        let new_order = new_order_msg.new_order;
+                        let result = self
+                            .process_new_order(new_order, new_order_msg.order_reason)
+                            .await;
                         new_order_msg.sender.send(result).await?;
                     }
                     TradingMessage::NewUser(new_user_msg) => {
+                        // todo(holzeis): shouldn't we also remove users from that set if they are
```

Comment:
Yes.

Reply:
I've created a ticket for that todo

https://github.com/get10101/10101/issues/1268